### PR TITLE
[NP-6952] Add no-jrl wizard test scenarios

### PR DESCRIPTION
**Pull request by KernelDeimos**
Adds support for wizard scenarios that can be rapidly modified and re-tested as they don't require clearing of journals

### CODE REVIEW

File: src/foam/u2/wizard/debug/TestWizardScenario.js [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+      adapt: function (o, n, prop) {
+        n = n.map(spec => foam.Array.isInstance(spec) ? {
+          sourceId: spec[0],
+          targetId: spec[1]
+        } : spec);
+        return foam.core.FObjectArray.ADAPT.value.call(this, o, n, prop);
+      }
+    }
+  ]
+});
```
Comment (tharmaman) [inline src/foam/u2/wizard/debug/TestWizardScenario.js]:
```suggestion
});

```

File: src/foam/u2/wizard/debug/scenarios.js [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+        ['Entry','BasicData.B'],
+        ['Entry','BasicData.C'],
+        ['Entry','MinMax.D'],
+        ['MinMax.D','Choice.D.A'],
+        ['MinMax.D','Choice.D.B'],
+        ['MinMax.D','BasicData.C']
+      ]
+    }
+  ]
+});
```
Comment (tharmaman) [inline src/foam/u2/wizard/debug/scenarios.js]:
```suggestion
});

```

**Review by tharmaman — APPROVED** [review]

File: src/foam/u2/wizard/debug/scenarios.js [hunk 3]
```diff
@@ -0,0 +1,63 @@
+/**
+ * @license
+ * Copyright 2022 The FOAM Authors. All Rights Reserved.
+ * http://www.apache.org/licenses/LICENSE-2.0
+ */
+
+foam.CLASS({
+  package: 'foam.u2.wizard.debug.scenarios',
+  name: 'MinMaxChoicePrereqLiftScenario',
```
Comment (kgrgreer) [inline src/foam/u2/wizard/debug/scenarios.js:9]: Why is the file named after the package instead of the model?

File: src/pom.js [hunk 4]
```diff
@@ -912,6 +912,9 @@ foam.POM({
     { name: "foam/u2/wizard/debug/DebugWAO",                                              flags: "web" },
     { name: "foam/u2/wizard/debug/DebugContextIntercept",                                 flags: "web" },
     { name: "foam/u2/wizard/debug/DebugWizardletView",                                    flags: "web" },
+    { name: "foam/u2/wizard/debug/TestWizard",                                            flags: "web" },
```
Comment (kgrgreer) [inline src/pom.js]: Wouldn't hurt to switch the flags: to "web&debug", would it?

File: src/foam/u2/wizard/debug/scenarios.js (hunk 3 above)
Comment (KernelDeimos) [reply src/foam/u2/wizard/debug/scenarios.js:9]: easier to add more scenarios - I could also see metaprogramming being used in here to create a number of similar scenarios

File: src/pom.js (hunk 4 above)
Comment (KernelDeimos) [reply src/pom.js]: What turns on the debug flag? Is it always enabled in development?

Comment (jlhughes) [reply src/pom.js]: I like the 'debug' flag idea.

**Pull request merged**